Sources/PointFree/Routers/RouteHelpers.swift

```diff
@@ -0,0 +1,14 @@
+import PointFreeRouter
+
+public let router = pointFreeRouter(
+  appSecret: Current.envVars.appSecret,
+  mailgunApiKey: Current.envVars.mailgun.apiKey
```

Comment:
this is kinda cool. the point free router is actually a function that takes some dependencies and then returns a `Router<Route>`.